README.md

```diff
@@ -29,9 +29,9 @@ irc.mozilla.org.
 
 ### Browser Extensions
 
-#### Firefox
+#### Firefox and Seamonkey
 
-PDF.js is built into version 19+ of Firefox, however one extension is still available:
+PDF.js is built into version 19+ of Firefox, however one extension is still available, in Seamonkey it needs to be manually installed:
```

Comment:
In firefox it is built in nowadays. It is not an installable extension. In Seamonkey, I load the URL given below in line 36, and it installs the xpi file as an extension, which I can then view in about:addons.
If you mention Chrome and Opera in this readme, why not Seamonkey?